src/get-started/flutter-for/ios-devs.md

```diff
@@ -5,467 +5,836 @@ description: Learn how to apply iOS developer knowledge when building Flutter ap
 
 Your iOS knowledge and skill set
-are highly valuable when building with Flutter,
+are highly valuable when building with Flutter, 
 because Flutter relies on the mobile operating system
 for numerous capabilities and configurations.
 Flutter is a new way to build UIs for mobile,
 but it has a plugin system to communicate
 with iOS (and Android) for non-UI tasks.
-If you're an expert in iOS development,
-you don't have to relearn everything to use Flutter.
+<!-- Add this sentence once published -->
+<!-- Learn more at [iOS and Apple hardware interactions with Flutter][] -->
 
 Flutter also already makes a number of adaptations
 in the framework for you when running on iOS.
 For a list, see [Platform adaptations][].
 
 This document can be used as a cookbook by jumping around
 and finding questions that are most relevant to your needs.
+Pieces of sample code are embedded within this document, 
+but the full working examples can be tested on DartPad or viewed on Github.
 
-## Views
+<!-- Embed intro to iOS video when published -->
 
-### What is the equivalent of a UIView in Flutter?
+{% comment %} Nav tabs {% endcomment -%}
+<ul class="nav nav-tabs" id="editor-setup" role="tablist">
+  <li class="nav-item">
+    <a class="nav-link active" id="swiftui-tab" href="#swiftui" role="tab" aria-controls="swiftui" aria-selected="true">SwiftUI</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" id="uikit-tab" href="#uikit" role="tab" aria-controls="uikit" aria-selected="false">UIKit</a>
+  </li>
+</ul>
 
-{{site.alert.secondary}}
-  How is react-style, or _declarative_,
-  programming different from the
-  traditional imperative style?
-  For a comparison, see [Introduction to declarative UI][].
-{{site.alert.end}}
+{% comment %} Tab panes {% endcomment -%}
+<div class="tab-content">
 
-On iOS, most of what you create in the UI is done using view objects,
-which are instances of the `UIView` class.
-These can act as containers for other `UIView` classes,
-which form your layout.
+<div class="tab-pane active" id="swiftui" role="tabpanel" aria-labelledby="swiftui-tab" markdown="1">
 
-In Flutter, the rough equivalent to a `UIView` is a `Widget`.
-Widgets don't map exactly to iOS views,
-but while you're getting acquainted with how Flutter works
-you can think of them as "the way you declare and construct UI".
 
-However, these have a few differences to a `UIView`.
-To start, widgets have a different lifespan: they are immutable
-and only exist until they need to be changed.
-Whenever widgets or their state change,
-Flutter’s framework creates a new tree of widget instances.
-In comparison, an iOS view is not recreated when it changes,
-but rather it's a mutable entity that is drawn once
-and doesn't redraw until it is invalidated using `setNeedsDisplay()`.
+## SwiftUI Overview
 
```

Comment:
Since you are deleting previous text, you need to add some new text here. You should have text between two headers.

Reply:
Okay I added a quick sentence to say what the section will discuss